docs/sources/tempo/introduction/architecture.md

```diff
@@ -93,6 +93,21 @@ The compactor is also responsible for expiring data after the retention period f
 Compactors run on scheduled frequent intervals to deal with data that is not compacted and has been stored by the Ingesters.
 Compaction takes into account the data stored for specific traces to minimize search space on queries.
 
+### Backend scheduler and worker
+
+The Backend Scheduler is a forward-looking re-architecture of the compaction process, with the goal of improving the determinism and removing the duplication present in the current compaction process. The Backend Scheduler is responsible for the scheduling and tracking jobs which are assigned to Backend Workers for processing. The Scheduler component, in combination with the Worker, will eventually replace the current Compactor component.
+
+The Worker connects to the Scheduler via gRPC to receive jobs for processing. Workers are responsible for executing jobs assigned to them by the Scheduler and updating the job status back to the Scheduler. These jobs currently include compaction and retention, but will likely include other kinds of jobs in the future.
+
+The workers currently have the additional responsibility of maintaining the blocklist for all tenants, which was previously handled by the Compactor. The determination of which tenants to poll is coordinated through the ring, just as it is with the Compactor.
```

Comment:
If we're using upper case Workers, do we need to do the same here?

Reply:
I've used lowercase for all components here.  Good call out.